docs/sources/faq.md

```diff
@@ -259,6 +259,10 @@ If you *really* need to change its setting, make sure to change it
 back to its [default value](https://www.debian.org/releases/stable/i386/ch05s03.html.en) 
 afterwards.
 
+### Why do I get `Connection reset by peer` when making a request to a service running in a container?
+
+Typically, this message is returned if the service is already bound to your localhost. As a result, requests coming to the container from outside are dropped. To correct this problem, change the service's configuration on your localhost so that the service accepts requests from all IPs. 
```

Comment:
(I'm not a maintainer, just a (rather fanatic) visitor of the Docker project, so I cannot lgtm :))

I think it may still need a bit attention; "on *your localhost*" is a bit confusing, I think. I'm thinking something along;

    this happens if a service is only listening on 'localhost' ..... to correct this, 
    configure the service to listen on all interfaces (e.g. 0.0.0.0)

English is not my primary language, and I'm in doubt if these are the best terms to use. Perhaps @SvenDowideit has some suggestions (if you don't mind this PR to be held up a bit)